<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateUserDialog</class>
 <widget class="QDialog" name="updateUserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改用户信息</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="infoLabel">
       <property name="font">
        <font>
         <family>PingFang SC</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>更新用户？的信息：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <family>PingFang SC</family>
          </font>
         </property>
         <property name="text">
          <string>旧信息</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <family>PingFang SC</family>
          </font>
         </property>
         <property name="text">
          <string>新信息</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="nameCheckBox">
           <property name="font">
            <font>
             <family>PingFang SC</family>
            </font>
           </property>
           <property name="text">
            <string>姓名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="telCheckBox">
           <property name="font">
            <font>
             <family>PingFang SC</family>
            </font>
           </property>
           <property name="text">
            <string>电话</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="ageCheckBox">
           <property name="font">
            <font>
             <family>PingFang SC</family>
            </font>
           </property>
           <property name="text">
            <string>年龄</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="maxBorrowNumbersCheckBox">
           <property name="font">
            <font>
             <family>PingFang SC</family>
            </font>
           </property>
           <property name="text">
            <string>最大借书量</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="oldNameLabel">
           <property name="font">
            <font>
             <family>PingFang SC</family>
            </font>
           </property>
           <property name="text">
            <string>oldName</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="oldTelLabel">
           <property name="font">
            <font>
             <family>PingFang SC</family>
            </font>
           </property>
           <property name="text">
            <string>oldTel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="oldAgeLabel">
           <property name="font">
            <font>
             <family>PingFang SC</family>
            </font>
           </property>
           <property name="text">
            <string>oldAge</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="oldNumLabel">
           <property name="font">
            <font>
             <family>PingFang SC</family>
            </font>
           </property>
           <property name="text">
            <string>oldNum</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLineEdit" name="newNameLineEdit">
           <property name="font">
            <font>
             <family>PingFang SC</family>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="newTelLineEdit">
           <property name="font">
            <font>
             <family>PingFang SC</family>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="newAgeSpinBox">
           <property name="font">
            <font>
             <family>PingFang SC</family>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="newMaxBorrowNumbersSpinBox">
           <property name="font">
            <font>
             <family>PingFang SC</family>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="font">
        <font>
         <family>PingFang SC</family>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>updateUserDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
